java/ql/lib/semmle/code/java/security/XmlParsers.qll

```diff
@@ -144,7 +144,7 @@ private predicate constantStringExpr(Expr e, string val) {
 }
 
 /** An expression that always has the same string value. */
-private class ConstantStringExpr extends Expr {
+class ConstantStringExpr extends Expr {
```

Comment:
Copy this to your qll file rather than expose it: the heuristic is bad enough that we probably don't want to publicly expose this difficult-to-explain partial solution.